src/main/java/seedu/address/model/tutorial/Tutorial.java

```diff
@@ -16,9 +16,8 @@
 public class Tutorial {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Subjects should only contain alphanumeric characters only.";
-    public static final String VALIDATION_REGEX = "^[a-zA-Z0-9]+$";
-
+            "Tutorial name should only contain alphanumeric characters and spaces, and it should not be blank.";
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
```

Comment:
Maybe we could check for a specific set of tutorials offered at the centre via enum